Makefile

```diff
@@ -35,6 +35,8 @@ ifeq ($(OR_EXEC), )
 ifeq ("$(wildcard /usr/local/openresty-debug/bin/openresty)", "")
 	@echo "ERROR: OpenResty not found. You have to install OpenResty and add the binary file to PATH before install Apache APISIX."
 	exit 1
+else
```

Comment:
What about echoing a warn message here?

Reply:
Done

Reply:
@tokers The warn message will be better, I've submitted.